compile/typedef.go

```diff
@@ -53,13 +55,17 @@ func compileTypedef(file string, src *ast.Typedef) (*TypedefSpec, error) {
 			Reason: err,
 		}
 	}
+	isLong := annotations["json.type"] == "Long"
+	isTimestamp := annotations["json.type"] == "Date"
 
 	return &TypedefSpec{
 		Name:        src.Name,
 		File:        file,
 		Target:      typ,
 		Annotations: annotations,
 		Doc:         src.Doc,
+		IsLong:      isLong,
```

Comment:
Instead of adding multiple booleans (which may result in future booleans being snuck into here too) could we use an enum instead? This would also ensure/make clear that isLong and isTimestamp are mutually exclusive.

Reply:
thoughts @abhinav ?